File: my_tickets_bot/bot/src/models/location.py
"""Модель место проведения"""
import dataclasses

from .city import City


@dataclasses.dataclass
class Location:
    """Модель места проведения"""
    location_id: int
    name: str | None = None
    address: str | None = None
    city: City | None = None
    url: str | None = None

File: my_tickets_bot/backend/src/model/location.go
package model

type Location struct {
	Id      int64  `db:"id" json:"id,omitempty"`
	Name    string `db:"name" json:"name,omitempty"`
	Address string `db:"address" json:"address,omitempty"`
	Url     string `db:"url" json:"url,omitempty"`
	City    City   `db:"city" json:"city"`
}
